CMakeLists.txt

```diff
@@ -75,7 +75,7 @@ add_definitions(-fPIC)
 # A grep of 'std::auto_ptr<' in the header files show 7 hits, and 13 hits in cpp files
 #
 IF (${DISTRO} STREQUAL "Ubuntu_17.04")
```

Comment:
The comment just above these lines should be reviewed to check if valid or obsolete.

Reply:
Fixed in b9173da